### DISCUSSION TITLE
Ability to assign multiple categories to a feed

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6191] [Q&A]
**Is your feature request related to a problem? Please describe.**
Just like a flat file system it would be nice if we could use multiple tags (in freshrss terminology: category) for a feed

**Describe the solution you’d like**
Ability to assign multiple categories to a field

**Describe alternatives you’ve considered**
A clear and concise description of any alternative solutions or features you’ve considered.

**Additional context**

**Comment by Alkarex** [comment]
Native support of multiple categories per feed is out of scope for now https://github.com/FreshRSS/FreshRSS/issues/647 
However, there are alternatives:
* you can make *user queries* returning the results from selected feeds, e.g. `f:1,2,3`
    * https://freshrss.github.io/FreshRSS/en/users/user_queries.html
    * https://freshrss.github.io/FreshRSS/en/users/10_filter.html
* you can assign multiple *user labels* to articles (manually or automatically)

**Reply by Worm359** [reply]
Hi! Im just started using FreshRSS and trying to understand the intended workflow. 
My question seems to be related to this topic, so I hope it’s appropriate to ask it here...
Is there a way to apply a label to all existing articles that match a filter or search, not just to new ones? 
For example, when I add a new feed, I may later realise that I want all articles from that feed to have certain labels. 
In Label management I can define filter rules for a label, but they seem to apply only to newly fetched articles, not the ones already loaded. 
Is there a built-in way to apply labels retroactively? Thanks!
